src/components/Sources.css

```diff
@@ -59,13 +59,17 @@ html[dir="rtl"] .sources-header-info {
 .folder,
 .domain,
 .file,
-.worker {
+.worker,
+.refresh,
+.add-button {
   fill: var(--theme-splitter-color);
 }
 
 .domain,
 .file,
-.worker {
+.worker,
+.refresh,
+.add-button {
```

Comment:
It will probably take a few days since I don't have time to work on it today anymore. Sorry.